metricbeat/module/kubelet/container/data.go

```diff
@@ -0,0 +1,72 @@
+package container
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/metricbeat/mb"
+	"github.com/elastic/beats/metricbeat/module/kubelet"
+)
+
+func eventMapping(content []byte) ([]common.MapStr, error) {
+	events := []common.MapStr{}
+	var summary kubelet.Summary
+
+	err := json.Unmarshal(content, &summary)
+	if err != nil {
+		return nil, fmt.Errorf("Cannot unmarshal json response: %s", err)
+	}
+
+	node := summary.Node
+	for _, pod := range summary.Pods {
+		for _, container := range pod.Containers {
+			containerEvent := common.MapStr{
+				mb.ModuleData: common.MapStr{
+					"pod": common.MapStr{
+						"name":      pod.PodRef.Name,
+						"namespace": pod.PodRef.Namespace,
+						"node":      node.NodeName,
+					},
+				},
+
+				"name":       container.Name,
+				"start_time": container.StartTime,
+
+				"cpu": common.MapStr{
+					"usage.nanocores":       container.CPU.UsageNanoCores,
```

Comment:
That should be `"usage": common.MapStr{ "nanocores" }`. The reason is otherwise it will break with elasticsearch 2.x. We often use `common.MapStr.Put(...) in these cases as it does it automatically, but not sure how we could use this here ...